src/schema/v2/artwork/__tests__/artwork.test.js

```diff
@@ -2297,6 +2297,40 @@ describe("Artwork type", () => {
         expect(data.artwork.hasCertificateOfAuthenticity).toBe(true)
       })
     })
+    it("is set to proper object when ertificate_of_authenticity and coa_from_authenticating_body are true", () => {
```

Comment:
#minor #non-blocking we are totally not consistent about this, but I find test files to be slightly more readable when there's a blank line between subsequent tests/blocks. (Usually when I'm in a file I try to update it!)

Also #superminor typo here `ertificate_of_authenticity`.

Reply:
OOOH thank you good call on both!